### PR TITLE
referencing things from core

### PR DESCRIPTION
**Pull request by dlockhart**
Noticed these while I was in here -- worth cleaning up.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3742,10 +3774,11 @@
       }
     },
     "d2l-course-image": {
-      "version": "github:Brightspace/course-image#a88d731b29c97049fa3a6adfd8308edbc08c5eb4",
+      "version": "github:Brightspace/course-image#f0f0a8adee413102f7a07444e4cafa199a8b94de",
```
Comment (dbatiste) [inline package-lock.json]: This was my update to use our requestIdleCallback polyfill.

File: package-lock.json [hunk 2]
```diff
@@ -3576,7 +3608,7 @@
       }
     },
     "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#0c102bc84f630d205abba5b81a9ddbeac7393529",
+      "version": "github:BrightspaceHypermediaComponents/activities#5fb12f05bd58ca1f52c28b7801617ab8b1f18a24",
```
Comment (dlockhart) [inline package-lock.json]: @AlexBedley FYI

File: package-lock.json [hunk 3]
```diff
@@ -3808,7 +3841,7 @@
       }
     },
     "d2l-enrollments": {
-      "version": "github:BrightspaceHypermediaComponents/enrollments#658dbc00872d8b113eda4d46d2cca143354a0e2f",
+      "version": "github:BrightspaceHypermediaComponents/enrollments#97722032486883be70cb4930350b045a344a933b",
```
Comment (dlockhart) [inline package-lock.json]: @AllanKerr, @iFun FYI

File: package-lock.json [hunk 4]
```diff
@@ -4034,7 +4067,7 @@
       }
     },
     "d2l-multi-select": {
-      "version": "github:BrightspaceUI/multi-select#c4a0e2fe1960140b8461814c3dd0f614d5b17763",
+      "version": "github:BrightspaceUI/multi-select#f2627e8330fd7e0f8c14f255fbca62e1fc62735c",
```
Comment (dlockhart) [inline package-lock.json]: @emil-furniss FYI

File: package-lock.json [hunk 5]
```diff
@@ -4158,7 +4191,7 @@
       "dev": true
     },
     "d2l-organizations": {
-      "version": "github:BrightspaceHypermediaComponents/organizations#aca07fad60997f655debe43a3cfc786c05aa7de7",
+      "version": "github:BrightspaceHypermediaComponents/organizations#ddcd30086d0c858756514de558b732f763024e52",
```
Comment (dlockhart) [inline package-lock.json]: @m6jones FYI

**Pull request merged**